From eagle@newdream.net Fri Nov 24 16:40:18 2006
Message-ID: <456790E4.9030102@newdream.net>
Date: Fri, 24 Nov 2006 16:40:04 -0800
From: Eagle Jones <eagle@newdream.net>
To: Greg Kroah-Hartman <gregkh@suse.de>
Subject: USB: airprime: add device id for dell wireless 5500 hsdpa card
Content-Type: text/plain; charset=ISO-8859-1; format=flowed

Added the device id (0x413c, 0x8115) for the Dell wireless HSDPA 5500,
which is a rebranded Novatel EU730.

Signed-off-by: Eagle Jones <eagle@newdream.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

---
 drivers/usb/serial/airprime.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/serial/airprime.c
+++ gregkh-2.6/drivers/usb/serial/airprime.c
@@ -21,6 +21,7 @@ static struct usb_device_id id_table [] 
 	{ USB_DEVICE(0x1410, 0x1110) }, /* Novatel Wireless Merlin CDMA */
 	{ USB_DEVICE(0x1410, 0x1430) },	/* Novatel Merlin XU870 HSDPA/3G */
 	{ USB_DEVICE(0x1410, 0x1100) }, /* ExpressCard34 Qualcomm 3G CDMA */
+	{ USB_DEVICE(0x413c, 0x8115) }, /* Dell Wireless HSDPA 5500 */
 	{ },
 };
 MODULE_DEVICE_TABLE(usb, id_table);
